src/PackageDependencyViewExtension/PackageDependencyViewExtension.csproj

```diff
@@ -0,0 +1,103 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{5E76AAB3-6302-473E-9655-081B53FB1419}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Dynamo.PackageDependency</RootNamespace>
+    <AssemblyName>PackageDependencyViewExtension</AssemblyName>
+    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>
+    <FileAlignment>512</FileAlignment>
+    <Deterministic>true</Deterministic>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>full</DebugType>
+    <Optimize>false</Optimize>
+    <OutputPath>..\..\bin\AnyCPU\Debug\</OutputPath>
```

Comment:
@QilongTang this project should not set it's output paths like this:

see all other csprojs import the output path as it's shared:

 <ImportGroup Label="PropertySheets">
    <Import Project="$(SolutionDir)/Config/CS.props" />
  </ImportGroup>

or something like this - for example, look at the notifications csproj - it does not have output paths.

Reply:
this then imports:
https://github.com/DynamoDS/Dynamo/blob/master/src/Config/CS.props#L7

Reply:
Good point, let me update